src/components/UserWallet.vue

```diff
@@ -45,16 +47,21 @@ type ModalState =
 const userStore = useUserStore()
 const modalStore = useModalStore()
 const rpcStore = useRpcStore()
+const appStore = useAppStore()
 const showModal: Ref<ModalState> = ref(false)
 const { currency } = storeToRefs(rpcStore)
 const chainSelectedForEdit: Ref<number | null> = ref(null)
 const showAddressListDropDown = ref(false)
 
 const currencyStore = useCurrencyStore()
 const walletBalance = computed(() => {
+  let DecimalPow = 18
```

Comment:
This should especially be genericized.

Reply:
Fixed here [ba5fee8](https://github.com/arcana-network/wallet-ui/pull/535/commits/ba5fee89321cfaab746e450f6d3411ae17261c15)

Reply:
Fixed here https://github.com/arcana-network/wallet-ui/pull/535/commits/ba5fee89321cfaab746e450f6d3411ae17261c15